### DISCUSSION TITLE
Добавить clickAction на HasMany

### DISCUSSION THREAD

**Discussion by Siesta** [moonshine-software/moonshine#949] [Ideas]
Не хватает возможности накинуть на HasMany clickAction

Если по-простому то в src/Fields/Relationships/HasMany.php методе tableValue на возврате не хватает простановки аттрибута 

```php
->customAttributes([
                'data-click-action' => $this->getResource()->getClickAction()
            ])
```

и в src/Buttons/HasManyButton.php методе for на возврате к примеру

```php
            ->customAttributes(['class' => 'edit-button'])
```

**Comment by lee-to** [comment]
Создал issue
